lib/bolt/plugin/prompt.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Bolt
+  class Plugin
+    class Prompt
+      def initialize
+        # Might not need this
+        @logger = Logging.logger[self]
+      end
+
+      def name
+        'prompt'
+      end
+
+      def hooks
+        ['lookup_value']
```

Comment:
If this hook is only going to be used inside a "config" block I think we should make it "inventory_config_lookup".